packages/ui-react/src/hooks/useFilteredContent.ts

```diff
@@ -0,0 +1,78 @@
+import type { Content } from '@jwp/ott-common/types/config';
+import { Breakpoint, getScreenSize } from '@jwp/ott-ui-react/src/hooks/useBreakpoint';
+
+const DEVICE_FILTER_LABELS: Record<string, string> = {
+  mobile: 'mobile',
+  tablet: 'tablet',
+  desktop: 'desktop',
+} as const;
+
+const COUNTRIES: Record<string, string> = {
+  DE: 'Germany',
+  FR: 'France',
+  NL: 'Netherlands',
+  RO: 'Romania',
+} as const;
+
+const TIMEZONES: Record<string, string> = {
+  'Europe/Berlin': 'DE',
+  'Europe/Paris': 'FR',
+  'Europe/Amsterdam': 'NL',
+  'Europe/Bucharest': 'RO',
+} as const;
```

Comment:
As an alternative we can also consider using `new Intl.DateTimeFormat().resolvedOptions().locale`

Reply:
That could be an alternative, but it might be a bit more tricky to implement. For instance, I get locale `en-GB` in my browser but timezone `Europe/Amsterdam` 🤔
I think the risk here is that while certain locales might be associated with specific timezones (en-US and America/New_York), there isn't a one-to-one mapping. A single country can have multiple timezones, and multiple countries might share a timezone 🤔

<img width="762" alt="Screenshot 2025-04-30 at 11 02 52" src="https://github.com/user-attachments/assets/03e08ae6-a632-44a2-af2f-cf4655c22ccf" />